sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -1198,7 +1198,7 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
 !         Average omega for the last point near the poles - moorthi
           if (j ==1 .or. j == jm) then
             tx1 = 1.0 / im
-!     write(*,*)' j=',j,' omgamax=',maxval(omga(:,j,:)),' omgamin=',minval(omga(:,j,:))
+!     write(0,*)' j=',j,' omgamax=',maxval(omga(:,j,:)),' omgamin=',minval(omga(:,j,:))
```

Comment:
@lipan-NOAA Please revert this change. We would like the no-fatal messages go to stdout instead of stderr.

Reply:
reverted it